src/package/packageVersion.ts

```diff
@@ -411,8 +411,9 @@ export class PackageVersion {
   }
 
   private static getPackage2VersionFields(connection: Connection): string[] {
-    const apiVersion = connection.getApiVersion();
-    return Package2VersionFields.filter((field) => (apiVersion > '60.0' ? true : field !== 'ValidatedAsync'));
+    return parseInt(connection.getApiVersion(), 10) > 60
```

Comment:
comparing strings via `>` works fine until we get to 100, after which `100.0` < `99.0` so use numbers when comparing numbers